web/src/components/DLP/OverviewTab.vue

```diff
@@ -122,8 +131,21 @@
         :style="`column-count: ${assetSummaryColumnCount};`"
         class="px-3 ml-2"
       >
+        <div
+          v-if="assetSummaryBeingComputed"
+          class="text-subtitle-2"
+        >
+          The assets summary is being computed, please wait.
+        </div>
+        <div
+          v-else-if="!assetSummary || !Object.keys(assetSummary).length"
+          class="text-subtitle-2"
+        >
+          Assets summary computation failed. This dandiset does not contain any assets.
```

Comment:
```suggestion
      <span class="font-italic font-weight-bold">
          This Dandiset does not contain any assets.
      </span>
```

I don't think this condition means the asset summary *failed*: if there are no assets, then there is no summary.

OTOH, if the asset summary actually fails for any reason, we ought to capture that as a specific status to report (e.g., so the user can click something to report the problem to us).

Finally, I added the bold and italic to match with "special" messages such as the "no funding information" message visible in your example.